src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-configs.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "General OpenTelemetry collector configurations"
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: Here are some general configurations for your collector.
```

Comment:
I saw a few issues with your metadescriptions. I think best practices for these, because they show up in search engine previews and such, is to include 'New Relic' in there and make them as efficient as possible. So I might do something like 'General configurations for New Relic's OpenTelemetry collector'. or something. I noticed the host-related collector doc's metadescription seemed to need more detail - didn't mention 'host'.

Reply:
@zuluecho9, thanks for catching that! I updated the metaDescription entries in every file in OpenTelemetry.